lms/static/sass/multicourse/_course_about.scss

```diff
@@ -113,6 +113,9 @@
             &:hover, &:focus {
               color: rgb(255,255,255);
             }
+            div {
```

Comment:
Remove this inline styling. As I believe that after using `<span>` you wont be needing this.